react-charts/sparkline-chart/step1-answer.md

```diff
@@ -1,4 +1,5 @@
-Please copy the following code into the app.css file, replacing all of the content there:
+Please copy the following code into the app.css file, replacing all of the
```

Comment:
backticks needed. Copy the following code into the `app.css` file, replacing all the content.